frontends/PyCDE/src/esi.py

```diff
@@ -231,12 +231,22 @@ def package(sys: System):
         shutil.copy(f, hw_src)
       shutil.copy(bin_dir / "driver.cpp", hw_src)
       shutil.copy(bin_dir / "driver.sv", hw_src)
-      shutil.copy(esi_inc_dir / "ESIPrimitives.sv", hw_src)
-      shutil.copy(esi_inc_dir / "Cosim_DpiPkg.sv", hw_src)
-      shutil.copy(esi_inc_dir / "Cosim_Endpoint.sv", hw_src)
+
+      # Are these still required? Verilator chokes on an internal error with
```

Comment:
If you don't use esi-cosim-runner (which end users won't), they are required. IIRC, I added an option to esi-cosim-runner to not include them implicitly.